content/docs/tasks/telemetry/distributed-tracing-lightstep/index.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Distributed Tracing With LightStep
+description: How to configure the proxies to send tracing requests to LightStep.
+weight: 11
+keywords: [telemetry,tracing,lightstep]
+---
+
+This task shows you how to configure Istio to collect trace spans and send them to LightStep.
```

Comment:
This task shows you how to configure Istio to collect trace spans and send them to LightStep [𝑥]PM.
[𝑥]PM lets you analyze 100% of unsampled transaction data from large-scale production software to produce meaningful distributed traces and metrics that help explain performance behaviors and accelerate root cause analysis. For more information, visit [LightStep](https://lightstep.com).
At the end of this task, trace spans will be sent from the proxies, to a LightStep [𝑥]PM Satellite pool, and then be
consumable in the web UI.